raft/rafttest/node.go

```diff
@@ -62,6 +64,22 @@ func (n *node) start() {
 				n.Node = nil
 				close(n.stopc)
 				return
+			case p := <-n.pausec:
+				recvms := make([]raftpb.Message, 0)
+				for p {
+					// TODO: locking around paused?
+					n.paused = true
+					select {
+					case m := <-n.iface.recv():
```

Comment:
The recv channel has a large buffer; unless we're worried about overflowing that buffer we could simplify this whole block to

```
case p := <-n.pausec:
  for p {
    p = <-n.pausec
  }
```

No need to read and save the messages at this level; just leave them in the channel.